### DISCUSSION TITLE
中英文、文档切换

### DISCUSSION THREAD

**Discussion by obiscr** [ppoffice/hexo-theme-icarus#1046] [Q&A]
请问这个翻译是怎么设置的？
![image](https://user-images.githubusercontent.com/28687074/157724449-3af1abef-25b7-4fce-afdd-c6034593f4d0.png)

**Comment by ppoffice** [comment]
手动加的

**Comment by obiscr** [comment]
我现在在 _post 创建了 **en** 和 **zh-CN**， 然后 en 和 zh-CN 下面的文章都设置了 `language :  xxx`，现在语言是可以切换了。但是
主页显示的文章数量翻倍了。不知道怎么解决。

而且，我感觉跟 官方文档的不太一样：
比如 官方文档：
英文： https://ppoffice.github.io/hexo-theme-icarus/uncategorized/getting-started-with-icarus/
中文：https://ppoffice.github.io/hexo-theme-icarus/uncategorized/icarus快速上手/

> URL 没有en 或者 zh-CN ，感觉是直接切换到了对应的文档。

但是我的是：
中文：
http://localhost:4000/2021/10/17/zh-CN/文章标题/
http://localhost:4000/2021/10/17/en/文章标题/

> URL 里面有en 或者 zh-CN ，文档的名字是一模一样的。
> 我的这种方式方式还有一个问题：上面也提到了。比如原来有9篇文章在 ，一旦开了多语言，主页显示文章的数量就翻倍了。
我估计是因为 en 下面有 9篇，zh-CN下面有9篇，加起来就导致翻倍了。暂时不清楚如何解决。

请问这个具体如何配置，有参考吗？

**Comment by ppoffice** [comment]
https://github.com/ppoffice/hexo-theme-icarus/tree/site/source/_posts

**Reply by obiscr** [reply]
谢谢！URL的问题，我等会看一下。

请问首页显示的文档总数量翻倍的问题（文章列表也是如此，有两种语言翻译，就显示两个一模一样的文章），请问这个有什么好办法解决嘛
